test/cmocka/src/math/auditory/ref_mel_filterbank_16_test1.h

```diff
@@ -0,0 +1,557 @@
+/* SPDX-License-Identifier: BSD-3-Clause
+ *
+ * Copyright(c) 2022 Intel Corporation. All rights reserved.
+ */
+
+/* This file was created with script ref_auditory.m */
```

Comment:
I'll do the same update as I did for matrix cmocka test vectors export. Would path "sof/test/cmocka/m" be a suitable for common Matlab language helper functions?

Reply:
Done now